### ISSUE TITLE
0818 모델 정확도

### ISSUE THREAD

**Issue by Haeinnnn** [Haeinnnn/geonganghaeGYM#5]
template vannilla cnn 에서 바꾼부분과 acc를 올려주세요
기본수정 : model.add(Dense(2, activation='softmax')) # 2로바꿈

).flow_from_directory(
      "obj/train",

).flow_from_directory(
      "obj/test",

**Comment by Haeinnnn** [comment]
기본수정만 했을때의 
loss= 2.2562613487243652
acc= 0.5070422291755676

**Comment by Haeinnnn** [comment]
에폭 100
loss= 0.0010209257015958428
acc= 1.0

**Comment by Haeinnnn** [comment]
고생하셨습니다
